### PR TITLE
[HAR-8051] created a folder for the tool

### PR DESCRIPTION
**Pull request by swamy-harness**

### CODE REVIEW

File: adam/versionPage.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<html>
+ <head>
+  <title>PHP Test</title>
```
Comment (gunascode) [inline adam/versionPage.php]: Lets change the Title to Harness Version Info

File: adam/versionPage.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+ </head>
+ <body>
+ <?php
+    define("QAENV", "qa.harness.io");
+    define("STGENV", "stage.harness.io");
+    define("PRDENV", "app.harness.io");
+    $env_details = array(QAENV, STGENV, PRDENV);
```
Comment (gunascode) [inline adam/versionPage.php]: Lets start array with Prod first, stage and QA.

File: adam/versionPage.php [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    function startTable()
+    {
+        for($i = 0 ; $i < 10 ; $i++){
+            print "<br>";
+        }   
+        echo "<table align = \"center\" border = \"1\">";
+        echo "<tr>";
+            echo "<td bgcolor=\"powderblue\"><b> Environment / Release Type </b></td>";
+            echo "<td bgcolor=\"powderblue\"><b> Backend Build # <b></td>";
+            echo "<td bgcolor=\"powderblue\"><b> UI Build # <b></td>";
```
Comment (gunascode) [inline adam/versionPage.php]: We can give environment lables like Production -< url for prod> , Stage - <url for stage> etc

File: adam/versionPage.php (hunk 1 above)
Comment (swamy-harness) [reply adam/versionPage.php]: Done

File: adam/versionPage.php (hunk 2 above)
Comment (swamy-harness) [reply adam/versionPage.php]: Done

File: adam/versionPage.php (hunk 3 above)
Comment (swamy-harness) [reply adam/versionPage.php]: Done

**Pull request merged**